RELEASING.md

```diff
@@ -42,28 +42,51 @@ During the week before release, the Release Champion makes changes in preparatio
 - Update [releaseVersions](https://github.com/adoptium/ci-jenkins-pipelines/blob/master/pipelines/build/regeneration/release_pipeline_generator.groovy#L6) with correct incoming release versions.
 - Update https://github.com/adoptium/mirror-scripts/blob/master/releasePlan.cfg with expected tag, detail see https://github.com/zdtsw/mirror-scripts/tree/issue/3167#skara-repos-and-processes
 
-Create release branch on below repositories:
+#### Branching message for build related repositories
 
+Post the below  message to the #build & #release channels in Slack:
+
+In Preparation for next weeks release, I'm proposing to branch the
+following repositories, in order that this branch becomes the baseline
+for the release. Shout now if you need to get any PRs merged for the release,
+as today is the last day, the following repositories will be branched:
+  ( temurin-build, ci-jenkins-pipelines, jenkins-helper).
+
+#### Create release branch on below repositories:
+ 
 - temurin-build <https://github.com/adoptium/temurin-build>
 - ci-jenkins-pipelines <https://github.com/adoptium/ci-jenkins-pipelines>
 - jenkins-helper <https://github.com/adoptium/jenkins-helper>
 
-`temurin-build` and `ci-jenkins-pipeline` share the same branch name,`jenkins-helper` might have a different branch name.
-These branches will require at least one approval to push commit to.
+These branches should be named according to the following format (vYYYY.MM.NN) ,e.g v2023.03.01 , whereby the final element is an incremental counter appended to the year and month of the release.
```

Comment:
The v-string looks a lot like a date. Could we change the last full-stop to a plus or a dash or something, to prevent misunderstandings? E.g. v2023.03+01 or v2023.03-01

Reply:
```suggestion
These branches should be named according to the following format (vYYYY.MM+NN) ,e.g v2023.03+01 , whereby the final element is an incremental counter appended to the year and month of the release.
```